ibexOverview.js

```diff
@@ -0,0 +1,140 @@
+var PORT = 60000;
+var HOST = "http://dataweb.isis.rl.ac.uk" 
+//$('.btn').button();
+
+function refresh() {
+	$.ajax({
+		url: HOST + ":" + PORT + "/",
+		dataType: 'jsonp',
+		data: {"Instrument": "all"},
+		timeout: 1000,
+		error: function(xhr, status, error){ 
+			window.alert("Error: JSON not read");
+		},
+		success: function(data){ 
+			//window.alert(JSON.stringify(data));
+			display_data(data);
+		}
+	});
+}
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function sortObject(o) {
+    var sorted = {},
+    key, a = [];
+
+    for (key in o) {
+        if (o.hasOwnProperty(key)) {
+            a.push(key);
+        };
+    };
+
+    a.sort();
+
+    for (key = 0; key < a.length; key++) {
```

Comment:
`key` is used as two different loop variables. Here it is an int, in the previous loop, it's a dictionary key. The usage here is as an index, not a key. A new index variable (e.g. `i`) should be used